.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor/pkg
-runc
+/runc
```

Comment:
A leading slash restricts a gitignore match to the top of the directory tree (relative to the gitignore file).

It can be a good idea to do this to avoid unintentionally matching other files later.